test/run_client_server_tests.py

```diff
@@ -22,17 +22,30 @@ def main():
     test_passed = {}
 
     # Start the test server
-    if not os.path.exists("c/server"):
+    serverName = "c/server"
+    if os.name == "nt":
```

Comment:
instead of this, let's start using os.path.join:

    serverName = os.path.join("c", "server")